util/config.dist.json

```diff
@@ -13,8 +13,7 @@
   },
   "inclusionConfig": {
     "interval": "630s",
-    "fetchBatchSize": 30,
-    "maxGetEntries": 30
+    "maxGetEntries": 30000
```

Comment:
I notice `util/config.dist.json` uses a `maxGetEntries` of 30,000, `util/config/ct-woodpecker.localdev.json` uses a value of 3,000 and `test/config/ct-woodpecker.docker.json` uses a value of 30. Was this intentional or should they all use the same value for consistency?

Reply:
Changed to be consistent. This was based on me changing my mind a few times about what would be a reasonable value for this. Certainly 30 is too low. Settled on 3000, with the idea that all the queries would complete in the span of a few seconds. However I'm now realizing our example intervals are too high and we should have a note about maxGetEntries * interval should be greater than the average growth in log size. One more revision incoming.